ios/musapreactnative/MusapModuleBridge.swift

```diff
@@ -50,42 +56,58 @@ class MusapModule: NSObject {
     }
 
   @objc
-  func sign(_ req: NSDictionary, callback: @escaping RCTResponseSenderBlock) {
+  func sign(_ req: NSDictionary, completion: @escaping RCTResponseSenderBlock) {
     do {
       let reqObj = try req.toSignatureReq()
       
       let musapCallback: (Result<MusapSignature, MusapError>) -> Void = { result in
         switch result {
         case .success(let musapSignature):
-          callback(["Data successfully signed: \(musapSignature.getB64Signature())"])
+          completion(["Data successfully signed: \(musapSignature.getB64Signature())"])
         case .failure(let error):
-          callback(["Error signing the data: \(error.localizedDescription)"])
+          completion(["Error signing the data: \(error.localizedDescription)"])
         }
       }
       
       Task {
         await MusapClient.sign(req: reqObj, completion: musapCallback)
       }
     } catch let error {
-      callback(["Error signing the data: \(error.localizedDescription)"])
+      completion(["Error signing the data: \(error.localizedDescription)"])
     }
   }
 
+  
   @objc
-  func listEnabledSscds(_ resolver: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) {
-    guard let sscds = MusapClient.listEnabledSscds() else {
-      reject("Error", "Unable to list enabled SSCDs", nil)
-      return
-    }
-    let sscdList = sscds.map { $0.toWritableMap() }
-    resolver(sscdList)
+  func enableSscd(_ sscdType: String) -> Any? { // -> Void crashes the app
+      let sscd: any MusapSscdProtocol
+      switch sscdType {
+      case "HSM":
+          sscd = SecureEnclaveSscd()
```

Comment:
Whut?

Reply:
Security enclave is not secure enclave. Why do I have to make a mention on a completely wrong name for something, mentioning the actual name for these things and then we still end up with a wrong/confusing name for it :(

Reply:
1st comment -> Sander named it HSM and I explained to him that HSM is an external hardware keystore, so we decided to use TEE which is the name that MUSAP uses to name the local phone keystores, otherwise we have to update the code in the front-end everytime we switch from android to iOS.

2nd comment ->  Not sure if I understand it, but SecureEnclaveSscd is a class in the musap-ios library and Apple calls it the same way: https://support.apple.com/guide/security/secure-enclave-sec59b0b31ff/web.

Reply:
Yes, so call things a secure enclave. Those are well established names. Don't start calling it a security enclave then.

The 2nd point, was that I made a comment about properly naming things. I mentioned the correct word Secure Enclave and then it gets changed into 'security enclave' before now being turned into TEE. If I say or ask something, the chances are pretty slim I would use the incorrect terminology, so then it becomes very frustrating if someone makes up their own name, whilst the correct term was handed over to them